### PR TITLE
Update setting-up.md

### PR DESCRIPTION
**Pull request by ghost**
Replacing hostname `mx.example.org` by `example.org` accordingly to the rest of the tutorial which doesn't refer to it (i.e. DNS configuration).

### CODE REVIEW

**Comment by foxcpp** [comment]
Do you think it might be better to use mx.example.org instead in the whole tutorial?

**Comment by ghost** [comment]
Hi @foxcpp. 
Yes maybe, but we'd have to change more things in the documentation :)

**Comment by foxcpp** [comment]
> Yes maybe, but we'd have to change more things in the documentation :)

Well, if it is better - why not?

**Comment by ghost** [comment]
New commit. Thanks in advance for your review.

File: docs/tutorials/setting-up.md [hunk 1]
```diff
@@ -110,8 +127,8 @@ own). Here is how your DNS zone should look like:
 example.org.   A     10.2.3.4
```
Comment (foxcpp) [inline docs/tutorials/setting-up.md:129]: Probably these need to be updated as well to mention mx1

Comment (ghost) [reply docs/tutorials/setting-up.md:129]: Done

**Comment by foxcpp** [comment]
Oh well, can you resolve that conflict?

**Comment by ghost** [comment]
Done.

File: docs/tutorials/setting-up.md [hunk 2]
```diff
@@ -144,13 +163,14 @@ at https://mta-sts.example.org/.well-known/mta-sts.txt:
...
 
-**Note**: mx.example.org in the file is your MX hostname, example.org in URL is
-the domain you are receiving messages for. In simple configurations, they are
-may be the same, but this is not the case for more complex setups.
-If you have multiple MX servers - add them all once per line, like that:
+**Note**: mx1.example.org in the file is your MX hostname, In a simple configuration,
+it will be the same as your hostname example.org.
+In a more complex setups, you would have multiple MX servers - add them all once
+per line, like that:
+=======
```
Comment (foxcpp) [inline docs/tutorials/setting-up.md]: Woops.

Comment (ghost) [reply docs/tutorials/setting-up.md]: What do you mean ?

Comment (foxcpp) [reply docs/tutorials/setting-up.md]: `======` probably should not be here.

**Review by foxcpp — APPROVED** [review]
According to Murphy's law, "Anything that can go wrong will go wrong".

You did not add AAAA record for MX domain and there is still a typo: "to easier" sound be "to ease".
But I'm going to merge that anyway since it is already taking way more time than it should.

**Pull request merged**